resources/css/bem/profile-badges.less

```diff
@@ -26,4 +26,8 @@
     height: 40px;
     object-fit: contain;
   }
+
+  &__date {
```

Comment:
Could use a gap for the date like `margin-top: 5px`, otherwise it's too close
![Screenshot 2023-09-21 at 20 11 22](https://github.com/ppy/osu-web/assets/1694544/7344e9c4-af7e-4c94-8b9f-584a3b95336a)
![image](https://github.com/ppy/osu-web/assets/1694544/b1d7616a-e448-4f7f-85b3-0f7a28ae8b0f)